osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -50,7 +50,7 @@ private double strainValueOf(DifficultyHitObject current)
             if (osuLastObj.BaseObject is Slider && withSliders)
             {
                 double movementVelocity = osuCurrObj.MovementDistance / osuCurrObj.MovementTime; // calculate the movement velocity from slider end to current object
-                double travelVelocity = osuCurrObj.TravelDistance / osuCurrObj.TravelTime; // calculate the slider velocity from slider head to slider end.
+                double travelVelocity = osuLastObj.TravelDistance / osuLastObj.TravelTime; // calculate the slider velocity from slider head to slider end.
```

Comment:
Nit: now with this change the code reads back to front chronologically a bit (current object is used before last). It looks correct, but I'd just swap order to follow more naturally.

Reply:
Funny. I originally had inverted these lines because of the same feeling, but reverted in order to reduce changes. Agree.